CHANGELOG

```diff
@@ -1,33 +1,315 @@
-# Changelog Since v0.6.0
```

Comment:
Can we keep the older changelogs?

Also maybe put the release version in the title, like
```
# v1.0.0: Changelog since v0.7.0
```

Or a different file per minor version, and keep them all in a CHANGELOG/ folder?

Reply:
Created a CHANGELOG folder following the https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG model.